veracode/submit_artifacts.py

```diff
@@ -363,7 +363,7 @@ def submit_artifacts(  # pylint: disable=too-many-return-statements, too-many-br
 
 
 @validate
-def is_ready_for_new_build(*, upload_api: UploadAPI) -> bool:
+def check_if_build_exists(*, upload_api: UploadAPI) -> bool:
```

Comment:
What about just `build_exists` since it returns a bool? Then you can have a very natural `if build_exists(upload_api=upload_api):`